lib/gitlab/ci/templates/pipeline/AndroidTemplate.yml

```diff
@@ -13,12 +13,16 @@ variables:
   DEFAULT_IMAGE: "jangrewe/gitlab-ci-android"
   DEPLOY_DEBUG_APK_SLACK_MESSAGE: "Hello Team! Here is the latest debug APK from branch `${CI_COMMIT_REF_NAME}`. It was triggered by: `${CI_PIPELINE_SOURCE}`."
   DEPLOY_DEBUG_APK_PATH: app/build/outputs/apk
+  DEPLOY_DEBUG_AAR_PATH: app/build/outputs/aar
   DEPLOY_DEBUG_APK_NAMES: yourdebugapkname
   DEPLOY_RELEASE_APK_SLACK_MESSAGE: "Hello Team! Here is the latest release APK triggered by tag: `${CI_COMMIT_TAG}`"
```

Comment:
Nit, maybe change to the below:

DEPLOY_RELEASE_SLACK_MESSAGE: "Hello Team! Here is the latest release APK/AAR triggered by tag: `${CI_COMMIT_TAG}`"